debian/control

```diff
@@ -53,7 +53,7 @@ Build-Depends: btrfs-tools,
                pkg-config,
                python (>= 2.6.6-3~),
                python-argparse | libpython2.7-stdlib,
-               python-dev,
+               python-all-dev,
```

Comment:
no, ${python:`<something>`} subst vars are generated by dh_python*, it by itself is a build dependency listed as "dh-python". so it's like an problem of egg-and-chicken or self-bootstrap. we can hardly let dpkg-buildpackage to figure out what the build-dependency is before the build-dependencies are ready.